src/models/model_DatasetFull.jl

```diff
@@ -2,27 +2,43 @@
 # Do not modify this file directly. Modify the OpenAPI specification instead.
 
 
-@doc raw"""DatasetFull
-
-    DatasetFull(;
-        name=nothing,
-        description=nothing,
-        group=nothing,
-        sequence=nothing,
-        dataTables=nothing,
-        geogLevels=nothing,
-        hasMultipleDataTypes=nothing,
-        breakdowns=nothing,
-    )
-
-    - name::String
-    - description::String
-    - group::String
-    - sequence::Int64
-    - dataTables::Vector{DataTableSimple}
-    - geogLevels::Vector{DatasetFullGeogLevelsInner}
-    - hasMultipleDataTypes::Bool
-    - breakdowns::DatasetFullBreakdowns
+"""
+```
+DatasetFull(;
+    name=nothing,
+    description=nothing,
+    group=nothing,
+    sequence=nothing,
+    dataTables=nothing,
+    geogLevels=nothing,
+    hasMultipleDataTypes=nothing,
+    breakdowns=nothing,
+)
+```
+This function creates a dataset given the description and the group of the data, the sequence, the geographical information and if it has multiple data types and breakdowns.
```

Comment:
Could you shorten this description some? I am not entirely sure what this function is doing... Thanks!